src/components/ReplayableVideoAsset.vue

```diff
@@ -92,6 +103,7 @@ export default {
       if (this.videoEnded) return this.$t('video.replay');
       return this.isPlaying ? this.$t('video.pause') : this.$t('video.play');
     },
+    ariaLabelledByContainer: ({ id }) => `${id}-custom-controls ${id}-alt`,
```

Comment:
The `${id}-alt` part should probably only be added if alt text was provided since the element it refers to won't be rendered for videos without alt text.

Reply:
Sure, I fixed it https://github.com/apple/swift-docc-render/pull/807/commits/0bcacf6169211a954cef55b2997ca28fc48c0a88